src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filterBySex = people.filter(person => person.sex === 'm');
```

Comment:
filterBySex sounds more like the name of a function than the result you want to store in the const.
Please fix everywhere

```suggestion
  const men = people.filter(person => person.sex === 'm');
```